src/routes/Edit.svelte

```diff
@@ -14,14 +14,48 @@
     onMount(async () => {
         ga("send", "pageview");
         ga("send", "event", "version", mermaidVersion, mermaidVersion);
-        fromUrl(params.data);
+
+        const key = "_mermaid_history_";
+        Array.prototype.push.apply(historyList, JSON.parse(localStorage.getItem(key) || "[]"));
```

Comment:
Just curious, Any reason for this instead of `historyList = JSON.parse(localStorage.getItem(key) || "[]"` ?